reference/data-node-bootstrapping.md

```diff
@@ -0,0 +1,130 @@
+and that it is properly set up to handle password-less logins from the
+access node. This means using either certificates or password files,
+as described in the [Data Node Authentication Chapter][distdb-auth].
+
+When bootstrapping the data node the access node will:
+
+1. Ensure that there is a database on the data node.
+2. Ensure that the extension is available in the database.
+3. Set metadata on the data node.
+
+## Setting up the database on the data node
+
+First, the access node has to ensure that there is a properly
+configured database on the data node by either bootstrapping a new
+database, or verifying that it is correctly set up, depending on the
+value of the `bootstrap` option to `add_data_node`.
+
+The database name on the data node is either the name of the database
+on the access node, or the value of the `database` option to
+`add_data_node`.
+
+### Bootstrapping the database
+
+If `bootstrap` is true (which is the default), the access node will
+first connect to either the `postgres` database or, if the `postgres`
+database does not exist, to the `template0` database and create the
+database:
+
+```
+CREATE DATABASE <database>
+   TEMPLATE template0 OWNER <owner>
+   ENCODING <encoding> LC_COLLATE <collation>
+   LC_CTYPE <character type>
+```
+
+Here, the *owner* is the user connected on the access node, the
+*encoding*, *collation*, and *character type* is taken from the
+current database on the access node.
+
+Once the database is created, the access node disconnect and continue
+with bootstrapping the extension. If the database already exists on
+the data node, this step is skipped.
+
+[Currently, there is no validation of the database as there is if
+`bootstrap` is false. IMHO, we should add that. /Mats]
+
+### Validation of the database
+
+If `bootstrap` is false, the database will be validated by checking
+that the encoding, character type, and collation is the same as the
+current database on the access node.
+
+## Setting up the extension
+
+The `timescaledb` extension need to be installed in the database on
+the data node, so the next step is ensuring that there is a valid
+extension installed in the database by either creating the extension,
+or verifying that the extension is compatible with the extension
+installed on the access node.
+
+### Bootstrapping the extension
+
+#### Creating the schema for the extension
+
+To install the extension, a schema is created to store the objects
+that the `timescaledb` extension needs. The schema name is decided by
+picking the same schema name as on the access node and using that, so
+if you have created the `timescaledb` extension in the default manner,
+`public` will be used for the schema on the access node, hence
+`public` will be used on the data node as well.
+
+If the schema name for the extension on the access node is `public`,
+no new schema is created, but otherwise it is expected that the schema
```

Comment:
What does it mean `expected`? Will it return error if the schema exists?

Reply:
Updated.